setup.cfg

```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = plotannot
+version = attr: plotannot._version.__version__
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/msbentsen/plotannot
+author = Mette Bentsen
+author_email = mette.bentsen@mpi-bn.mpg.de
+license = MIT
+license_file = LICENSE
+platforms = Linux, Mac OS X, Windows
+classifiers =
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Topic :: Scientific/Engineering :: Visualization
+    Programming Language :: Python :: 3
+    Framework :: Matplotlib
+
+[options]
+packages = find:
```

Comment:
alternatively could explicitly list `plotannot` but this should have the same result.